demo/Dockerfile

```diff
@@ -22,7 +22,6 @@ RUN --mount=type=cache,target=/root/.composer/cache \
         drupal/facets:^1.3 \
         drupal/matomo:^1.7 \
         drupal/pdf:1.x-dev \
-        drupal/rdfui:^1.0-beta1 \
```

Comment:
I assume this is fine, just thought I'd bring attention to it if it was intended for another branch or commit.